diablo/api/course_controller.py

```diff
@@ -288,3 +285,6 @@ def _after_approval(course):
                 all_approvals=approvals,
                 course=course,
             )
+        return SisSection.get_course(section_id=course['sectionId'], term_id=course['termId'])
+    else:
+        return course
```

Comment:
If course is updated in approval-post-processing then give the front-end a fresh course instance.